test/unit/test-helpers.js

```diff
@@ -39,4 +39,4 @@ var TestHelpers = {
   }
 };
 
-export default TestHelpers;
\ No newline at end of file
+export default TestHelpers;
```

Comment:
Not sure why it's diff'ing here; does my IDE need some kind of formatter on this line?